rocky/reports/templates/partials/report_ooi_list.html

```diff
@@ -103,7 +110,7 @@ <h2>
         {% endif %}
     </div>
 </section>
-<form id="select_all_objects_form" class="inline" method="get">
+<form novalidate id="select_all_objects_form" class="inline" method="get">
```

Comment:
Is this 'novalidate' needed?

Reply:
We put here `novalidate` to show form error wrapped in Manon styling, like for example:
<img width="1243" alt="image" src="https://github.com/user-attachments/assets/fb677131-21ff-48b0-8939-98f42e7b37c0">